### PR TITLE
add exercise 1

### PR DESCRIPTION
**Pull request by nguyenvanman**

### CODE REVIEW

File: Exercise_1/Resources/layout/Main.axml [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    <LinearLayout
+        android:orientation="horizontal"
+        android:minWidth="25px"
+        android:minHeight="25px"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:id="@+id/linearLayout2"
+        android:background="@color/Gainsboro">
+        <TextView
+            android:text="I agree to the "
```
Comment (voquanghoa) [inline Exercise_1/Resources/layout/Main.axml]: Move string to `Strings.xml`

File: Exercise_1/Resources/layout/Main.axml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            android:paddingLeft="@dimen/paddingSmall"
+            android:textColor="@android:color/black"
+            android:textSize="15sp" />
+        <Button
+            android:text="Terms of Service"
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:id="@+id/button3"
+            android:background="@null"
+            android:textSize="15sp"
```
Comment (voquanghoa) [inline Exercise_1/Resources/layout/Main.axml]: Move to `dimens.xml`

File: Exercise_1/Resources/layout/Main.axml (hunk 1 above)
Comment (nguyenvanman) [reply Exercise_1/Resources/layout/Main.axml]: Done

File: Exercise_1/Resources/layout/Main.axml (hunk 2 above)
Comment (nguyenvanman) [reply Exercise_1/Resources/layout/Main.axml]: Done

**Pull request merged**